docs/notebooks/analysis/swept_langmuir/find_floating_potential.ipynb

```diff
@@ -0,0 +1,390 @@
+    "    1. [Notes about usage](#Notes-about-usage)\n",
+    "    1. [Knobs to turn](#Knobs-to-turn)\n",
+    "1. [Calculate the Floating Potential](#Calculate-the-Floating-Potential)\n",
+    "    1. [Interpreting results](#Interpreting-results)\n",
+    "    1. [Plotting results](#Plotting-results)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## How `find_floating_potential()` works\n",
+    "\n",
+    "1. The passed current array is scanned for points that equal zero and point-pairs that straddle $I = 0$.  This forms an collection of \"crossing-points.\"\n",
+    "1. The crossing-points are then grouped into \"crossing-islands\" based on the `threshold` keyword.\n",
+    "    - A new island is formed when a successive crossing-point is more (index) steps away from the previous corssing-point than defined by `threshold`.  For example, if `threshld=4` then an new island is formed if a crossing-point candidate is more than 4 steps away from the previous candidate.\n",
+    "    - If multiple crossing-islands are identified, then the function will compare the total span of all crossing-islands to `min_points`.  If the span is greater than `min_points`, then the function is incapable of identifying $V_f$ and will return `numpy.nan` values; otherwise, the span will form one larger crossing-island.\n",
+    "1. To calculate the floating potential...\n",
+    "    - If the number of points that make up the crossing-island is less than `min_points`, then each side of the \"crossing-island\" is equally padded with the nearest neighbor points until `min_points` is satisfied.\n",
+    "    - If `fit_type=\"linear\"`, then a `scipy.stats.linregress` fit is applied to the points that make up the crossing-island.\n",
+    "    - If `fit_type=\"exponential\"`, then a `scipy.optimize.curve_fit` fit is applied to the points that make up the crossing-island."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Notes about usage\n",
+    "\n",
+    "- The function provides no signal processing, so, if needed, the user must smooth, sort, crop, etc. the arrays before passing to the function.\n",
+    "- The function requires the voltage array to be monotonically increasing or decreasing.\n",
+    "- If the total range spanned by all crossing-islands is less than or equal to `min_points`, then `threshold` is ignored and all crossing-islands are grouped into one island."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Knobs to turn\n",
+    "\n",
+    "- `fit_type`\n",
+    "\n",
+    "    There are two types of curves that can be fitted to the identified crossing point data, `\"linear\"` and `\"exponential\"`.  The former will fit fit a line to the data, whereas, the later will fit and exponential curve with an offset.  The default curve is `\"exponential\"` since swept Langmuir data is not typically linear as it passes through $I=0$.\n",
```

Comment:
```suggestion
    "    There are two types of curves that can be fitted to the identified crossing point data: `\"linear\"` and `\"exponential\"`.  The former will fit fit a line to the data, whereas, the later will fit and exponential curve with an offset.  The default curve is `\"exponential\"` since swept Langmuir data is not typically linear as it passes through $I=0$.\n",
```